Anabaena/index.html

```diff
@@ -0,0 +1,104 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <script
+      language="javascript"
+      type="text/javascript"
+      src="../libraries/p5.min.js"
+    ></script>
+    <script type="module">
+      import { sketch } from "./Anabaena.js";
+      import "../sketchlib/sketchbook-navbar.js";
+      const p5_sketch = new p5(sketch, "sketch");
+    </script>
+    <link rel="stylesheet" href="../styles/layout.css" />
+    <script
+      id="MathJax-script"
+      async
+      src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-mml-chtml.js"
+    ></script>
+  </head>
+  <body>
+    <sketchbook-navbar></sketchbook-navbar>
+    <div class="container">
+      <h1>Anabaena Catenula (2025)</h1>
+      <div id="sketch"></div>
+      <div class="description">
+        <p>
+          Move the mouse (desktop) or touch and drag the canvas (mobile) to move
+          the bacterium around.
+        </p>
+        <p>
+          This sketch is a combination of the animation technique used in the
+          <a href="../Worm/">Worm</a> sketch, but also uses an L-system to
+          simulate bacterial growth.
+        </p>
+        <p>
+          The L-system I'm using is based on the L-system described in
```

Comment:
Also, mention one key feature of this L-system: it corresponds to fibonacci growth!